style.css

```diff
@@ -1,17 +1,26 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Space+Mono&display=swap');
 
+* {
+  padding: 0;
+}
```

Comment:
- [ ] When I look at your design it looks great to me. However, there is some extra spacing around it. 
As you can see on this image:

<img width="373" alt="Screen Shot 2021-11-28 at 12 02 22 AM" src="https://user-images.githubusercontent.com/69011963/143719123-b19d7a3c-5d17-4ffe-9fdd-ba2f3582a2e6.png">
It should be full-filled, with no spacing around it. For fixing this issue you are free to accept my suggestions...


```suggestion
* {
  padding: 0;
  margin:0;
}
```